site/en/docs/extensions/mv3/screen_capture/index.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: "layouts/doc-post.njk"
+title: "Audio recording and screen capture"
+seoTitle: "Chrome Extensions: Audio recording and screen capture"
+date: 2023-04-14
+updated: 2023-04-14
+description: How to record audio or video from a tab, window or screen
+---
+
+APIs such as [chrome.tabCapture][tabcapture] and [getDisplayMedia][get-display-media] provide a
+variety of ways to monitor and record audio and video from a tab, window or screen.
+
+## Common Use Cases
```

Comment:
I think it'd be good to mention example application for different use cases, e.g. "for screen recording, e.g. in a video call, ..."

Reply:
For some of these I struggled to think of easily understandable examples (e.g I don't think it's helpful to say "If you're building a volume booster...", even though that's a reasonable use case). I might pass on this one unless you have thoughts.